### PR TITLE
added DevOps category and Vamp

### PR DESCRIPTION
**Pull request by gggina**

### CODE REVIEW

**Comment by mpark** [comment]
```
commit bb965cf48ad742ef09ef7c1019db7c221139ecd3
Author: Gina <ghofstede17@gmail.com>
Date:   Tue Jul 5 18:07:30 2016 +0200

    Added DevOps category and Vamp.

    This closes #127
```

File: docs/frameworks.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ layout: documentation
 ---
 # Software Projects Built on Mesos
 
+## DevOps tooling
+
+* [Vamp](http://http://vamp.io) is a deployment and workflow tool for container orchestration systems, including Mesos/Marathon. It brings canary releasing, A/B testing, auto scaling and self healing through a web UI, CLI and REST API.
```
Comment (haosdent) [inline docs/frameworks.md]: The path incorrect here?

Comment (haosdent) [inline docs/frameworks.md]: I post a quick fix at https://reviews.apache.org/r/49641/

**Comment by gggina** [comment]
Thanks.
How does this get added to http://mesos.apache.org/documentation/latest/frameworks/ ?

**Comment by jmlvanre** [comment]
It will get added to the website the next time it gets published.
We do that pretty regularly.

**Pull request closed without merging**